api/db-cdr/csv/achilles_analysis.csv

```diff
@@ -10,18 +10,25 @@ analysis_id,"analysis_name","stratum_1_name","stratum_2_name","stratum_3_name","
 12,"Number of persons by race and ethnicity","race_concept_id","ethnicity_concept_id","","","","pie","counts"
 603,"Number of distinct procedure occurrence concepts per person","","","","","","column","counts"
 606,"Age distribution by procedure_concept_id","procedure_concept_id","gender_concept_id","","","","box","distribution"
-1815,"Distribution of numeric values, by measurement_concept_id and unit_concept_id","","","","","","column","counts"
+1806,"Distribution of age by measurement_concept_id","measurement_concept_id","gender_concept_id","null","null","null","box","distribution"
+1815,"Distribution of numeric values, by measurement_concept_id and unit_concept_id","measurement_concept_id","unit_concept_id","","","","box","distribution"
 1816,"Distribution of low range, by measurement_concept_id and unit_concept_id","","","","","","column","counts"
 1817,"Distribution of high range, by observation_concept_id and unit_concept_id","","","","","","column","counts"
+1900,"Measurement response gender distribution","measurement_concept_id","gender_concept_id","","value","unit_source_value","column","counts"
+1901,"Measurement response age distribution","measurement_concept_id","age_decile","","value","unit_source_value","column","counts"
+1911,"Measurement Male Distribution","measurement_concept_id","gender_concept_id",NULL,"value","unit_source_value","column","counts"
```

Comment:
We don't want the male and female analysis anymore.

Reply:
Done. Just added achilles result list response with list of male and female results to measurement analysis object.